feed.go

```diff
@@ -253,7 +253,7 @@ func (f *baseFeed) GetFollowings(limit int, offset int) ([]FeedID, error) {
 		outputFeeds []FeedID
 	)
 
-	endpoint := "feed/" + f.FeedSlug + "/" + f.UserID + "/" + "following" + "/"
+	endpoint := "feed/" + f.FeedSlug + "/" + f.UserID + "/" + "follows" + "/"
```

Comment:
According to the REST docs it's correct, isn't it? https://getstream.io/docs_rest/#follows